cmake/SpectreSetupPythonPackage.cmake

```diff
@@ -249,6 +243,17 @@ function(SPECTRE_PYTHON_ADD_MODULE MODULE_NAME)
       "${MODULE_LOCATION}/${PYTHON_FILE_JUST_NAME}"
       )
   endforeach(PYTHON_FILE ${ARG_PYTHON_FILES})
+
+  # Create empty __init__.py files if none exist
+  # We walk up the tree until we get to ${SPECTRE_PYTHON_PREFIX}
+  set(CURRENT_MODULE ${MODULE_LOCATION})
```

Comment:
Are these written into the source tree or the build directory? I found when we switched from auto `__init__.py` to manual `__init__.py` in the source tree it was annoying to deal with switching branches. Should we have a bash script that adds these and checks for them in PRs?

Reply:
Build directory! So this shouldn't be a problem.

The branch switching back then was only annoying if you had `PY_DEV_MODE` enabled, because then the init.py files are symlinks.

Reply:
So it shouldn't be a problem even in `PY_DEV_MODE` now?

Reply:
Not unless you have a branch where you add an init.py file in an intermediate directory in the Py package, like in PointwiseFunctions/.

Reply:
I'm fine with this, but just to clarify your last statement: If I add new init files in the _src_ tree, then they may clash with the auto-generated ones in the _build_ dir?

Reply:
Actually on second thought I think the issue with init.py files can't happen anymore at all because CMake doesn't overwrite the file anymore (there's an `EXISTS` check).